postprocess/crossplane.jsonnet

```diff
@@ -0,0 +1,26 @@
+/**
+ * Remove generated crossplane manifests if not required
+ */
+local com = import 'lib/commodore.libjsonnet';
+
+local inv = com.inventory();
+local params = inv.parameters.appcat;
+local cms = params.clusterManagementSystem;
+local file_extension = '.yaml';
+
+local isSingleCluster = cms.controlPlaneCluster && cms.serviceCluster;
```

Comment:
can't you import the vars file here as well?

Reply:
I had a problem here with importing the `vars.jsonnet`

Reply:
I will move this change in the next PR when I resolve the importing issues.